docs/04-shortcuts-settings.md

```
@@ -0,0 +1,331 @@
+  
+  // شفافیت پیشنهادات
+  "cursor.ui.suggestionOpacity": 0.7
+}
+```
+
+## سفارشی‌سازی میانبرها
+
+### نحوه تغییر میانبرها
+
+1. **باز کردن Keyboard Shortcuts:**
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)